common/lib/xmodule/xmodule/modulestore/split_mongo/split_draft.py

```diff
@@ -198,6 +198,8 @@ def delete_item(self, location, user_id, revision=None, skip_auto_publish=False,
                 branches_to_delete = [ModuleStoreEnum.BranchName.published, ModuleStoreEnum.BranchName.draft]
             elif revision is None:
                 branches_to_delete = [ModuleStoreEnum.BranchName.draft]
+                if location.category in DIRECT_ONLY_CATEGORIES:
```

Comment:
See my note to Adam about the actual structure of the about videos. TL; DR: they're actually HTML blocks with a youtube ID embedded in them, not a video module.